File: par/model/writeVector2mtx.m
function writeVector2mtx(filename,vector)

n=size(vector,1);

fileID = fopen(filename,'w');
fprintf(fileID,['%%']);
fprintf(fileID,['%%MatrixMarket vector array real general\n']);
fprintf(fileID,[num2str(numel(vector)), ' ', num2str(1)]);
fprintf(fileID,'\n');
fprintf(fileID,'%f \n',[vector]);
fclose(fileID);

end

File: par/common/writeVector2mtx.py
def writeVector2mtx(filename, array):

    fileID = open(filename, 'w')
    fileID.write('%%MatrixMarket vector array real general\n')
    fileID.write(str(len(array))+' 1\n')
    fileID.write('\n'.join(map(str, array)))
    fileID.write('\n')
    fileID.close()

